pywbem/_subscription_manager.py

```diff
@@ -565,17 +590,38 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True,
               :term:`RFC6874`, supporting ``-`` (minus) for the delimiter
               before the zone ID string, as an additional choice to ``%25``.
 
-            Note that the port is required in listener URLs.
+            Note that the port is required in listener URL.
```

Comment:
Lets take off the words "Note that".

Reply:
DONE as proposed.

I used that phrase because the syntax already shows the port as required, so I thought this sentence just needs to point that rule out, but it does not represent the rule as such.

However, turning that into a rule even when it repeats the rule shown in the syntax is not a mistake, certainly.